Project/PrivateVault App/Code/Model/CanBeNestedInFolder.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CanBeNestedInFolder.swift
+//  PrivateVault
+//
+//  Created by Elena Meneghini on 11/08/2021.
+//
+
+protocol CanBeNestedInFolder {
+	func belongsToFolder(_ folder: Folder) -> Bool
```

Comment:
I think these methods should be renamed to better follow Swift's naming conventions:

`belongs(to folder: Folder) -> Bool`
`add(to folder: Folder)`
`remove(from folder: Folder)`

Reply:
good idea, this looks much better